pyproject.toml

```diff
@@ -0,0 +1,121 @@
+[tool.poetry]
```

Comment:
I think we should clarify the poetry workflow in the documentation. Ex:
```
cd ~/udata && poetry install
cd ~/udata-front && poetry install
cd  ~/udata && poetry run inv serve
```
I had to remove my `udata.egg-info` locally for it to work, maybe we should add it in the Changelog for other udata users?